scraper/README.md

```diff
@@ -0,0 +1,52 @@
+# PermaplanT Scraper
+
+## Requirements
+
+-   nodejs 19.4.0
+-   npm
+
+## Installation and Usage
+
+1. Install dependencies
+
+```shell
+npm install && mkdir data
```

Comment:
data is already there but it should be noted that .env needs to be copied/adjusted.

```suggestion
npm install
```